### DISCUSSION TITLE
Filter list with "Diff-Path" but without "Diff-Expires:"

### DISCUSSION THREAD

**Discussion by Procyon-b** [uBlockOrigin/uBlock-issues#3282] [General]
I have 2 lists (2 adguard regional lists ; NL FR) that have a "_Diff-Path_" header without a "_Diff-Expires:_". uBO (_1.58.1b10_) handles them with no differential update: full 5 days update cycle. (I have only updated to _b11_ right now)

Should uBO use a default value for "_Diff-Expires:_" if there is only a "_Diff-Path_" header, or should adguard fix the headers in its lists ?

**Comment by garry-ut99** [comment]
Hmm, it seems not only NL, FR, but all AdGuard lists in uBO lack "Diff-Expires:":
- AdGuard – Ads
- AdGuard – Mobile Ads
- AdGuard Tracking Protection
- AdGuard URL Tracking Protection
- AdGuard – Annoyances

Clock icon says they were updated several days ago.

**Comment by gorhill** [comment]
AdGuard uses a different way to encode `diff-expires`, there is no code yet in uBO to deal with AdGuard's differential update approach.
